cmd/atlas/internal/url/provider.go

```diff
@@ -0,0 +1,52 @@
+package url
```

Comment:
I'd avoid using the word `url` as a package, because it's widely used as an std package. Also, keep it simple (for now) and move this logic to a single file under the `cmd/atlas` package.

Reply:
👍